### PR TITLE
feat: added better error handling

### PR DESCRIPTION
**Pull request by anuraghazra**

### CODE REVIEW

**Comment by anuraghazra** [comment]
Preview demo:

Error

Error

Error

Error

Error

File: api/index.js [hunk 1]
```diff
@@ -49,7 +52,11 @@ async function fetchStats(username) {
     totalStars: 0,
     contributedTo: 0,
   };
-  if (res.data.error) return stats;
```
Comment (anuraghazra) [inline api/index.js:52]: I did res.data.error previously which was basically useless lol

File: api/index.js [hunk 2]
```diff
@@ -67,12 +74,11 @@ async function fetchStats(username) {
...
 const createTextNode = (icon, label, value, lheight) => {
+  const classname = icon === "★" && "star-icon";
   return `
     <tspan x="25" dy="${lheight}" class="stat bold">
-    <tspan class="icon ${
-      icon === "★" && "star-icon"
-    }" fill="#4C71F2">${icon}</tspan> ${label}:</tspan>
-    <tspan x="155" dy="0" class="stat">${value}</tspan>
+    <tspan class="icon ${classname}" fill="#4C71F2">${icon}</tspan> ${label}:</tspan>
+    <tspan x="155" dy="0" class="stat">${kFormatter(value)}</tspan>
```
Comment (anuraghazra) [inline api/index.js:81]: kFormatted the values

**Comment by nombrekeff** [comment]
Nice addition!

**Pull request merged**